docs/setup/arch/external_storage/README.md

```diff
@@ -12,7 +12,7 @@ The S3 storage can be either on-premises or cloud-based. In any case it is requi
 
 Uploading data from the extern of the HPC infrastructure requires users to apply the [multipart upload protocol](https://docs.aws.amazon.com/AmazonS3/latest/userguide/mpuoverview.html), therefore the data file shall be divided into parts. The size limit of each part is defined in the response of FirecREST to the upload call. 
 
-Tha maximum size of the parts can be configured via the [`max_part_size parameter`](../../../setup/conf/README.md#bucket_lifecycle_configuration). The specified value must be lower than [S3's maximum limit](https://docs.aws.amazon.com/AmazonS3/latest/userguide/qfacts.html) of 5GB per part and it is applied to any upload data call.
+Tha maximum size of the parts can be configured via the [`max_part_size parameter`](../../../setup/conf/README.md#bucket_lifecycle_configuration). The specified value must be lower than [S3's maximum limit](https://docs.aws.amazon.com/AmazonS3/latest/userguide/qfacts.html) of 5GB per part and it is applied to any upload data call. When setting this value, please consider that clients using this protocol to upload parts may need to store them in memory before streaming the data to S3. Therefore, setting a large value for this parameter could impact clients utilizing FirecREST for data uploads. S3 limits the file uploads to [5TB](https://docs.aws.amazon.com/AmazonS3/latest/userguide/qfacts.html) with a maximum of [10,000 parts](https://docs.aws.amazon.com/AmazonS3/latest/userguide/qfacts.html). To fully utilize the maximum number of parts for transferring the largest allowable file, each part should be approximately 525 MB. FirecREST's default is set to 1GB per part.
```

Comment:
2 links should change in this file:
- `[bucket_lifecycle_configuration](../../../setup/conf/README.md#bucketlifestyleconfiguration)`
- `[max_part_size parameter](../../../setup/conf/README.md#multipartupload)`